blockbot-api/src/main/java/com/github/quiltservertools/blockbotapi/BlockBotApi.java

```diff
@@ -13,12 +11,14 @@
 import java.util.Set;
 
 public class BlockBotApi implements ModInitializer {
-    public static final Logger LOGGER = LogManager.getLogger();
+    public static final Logger LOGGER = LogManager.getLogger("BlockBot API");
     private static final Set<Bot> bots = new HashSet<>();
 
     @Override
     public void onInitialize() {
+        LOGGER.info("Initializing BlockBotAPI");
 
+        ServerLifecycleEvents.SERVER_STARTING.register(server -> LOGGER.info("Loaded " + bots.size() + " bots"));
```

Comment:
Bot's are probably going to be registered *after* initialize so this won't really work. It also already sends a message when a bot is registered.
```suggestion
```